tutorials/frontend/from_tflite.py

```diff
@@ -29,20 +29,32 @@
 
     pip install flatbuffers --user
 
+
 To install TFlite packages, you could use our prebuilt wheel:
 
 .. code-block:: bash
 
     # For python3:
-    wget https://raw.githubusercontent.com/dmlc/web-data/master/tensorflow/tflite/whl/tflite-0.0.1-py3-none-any.whl
-    pip install tflite-0.0.1-py3-none-any.whl --user
+    wget https://raw.githubusercontent.com/dmlc/web-data/master/tensorflow/tflite/whl/tflite-1.13.1-py3-none-any.whl
+    pip3 install tflite-1.13.1-py3-none-any.whl --user
 
     # For python2:
-    wget https://raw.githubusercontent.com/dmlc/web-data/master/tensorflow/tflite/whl/tflite-0.0.1-py2-none-any.whl
-    pip install tflite-0.0.1-py2-none-any.whl --user
+    wget https://raw.githubusercontent.com/dmlc/web-data/master/tensorflow/tflite/whl/tflite-1.13.1-py2-none-any.whl
+    pip install tflite-1.13.1-py2-none-any.whl --user
+
+
+or you could clone FrozenGene/tflite repo which contains generated tflite python module
```

Comment:
I think it is better to let users do it themselves. In my opinion, we should restrict the repo under the umbrella of dmlc.

Reply:
line 57 still shows the option where people can generate tflite themself.
Options are:
* install `tflite` using pip
* simply clone `tflite` repo
* generate `tflite` using `flatc`

Reply:
Yes. I saw it. My previous meaning is we restrict the repo url all are under the `dmlc`, not non-official ( for example FrozenGene/tflite.). However, I couldn't make sure whether we should have this restriction. Need more input from @tqchen @srkreddy1238 @yzhliu @kevinthesun  One solution maybe is we upload these generated python files to dmlc/web-data. Or just let users do it themselves like line 57.

Reply:
ok, I removed cloning `tflite` repo option from the tutorial.